llama-cpp-2/Cargo.toml

```diff
@@ -15,8 +15,12 @@ tracing = { workspace = true }
 
 [features]
 cublas = ["llama-cpp-sys-2/cublas"]
+metal = ["llama-cpp-sys-2/metal"]
 sampler = []
 
+[target.'cfg(all(target_os = "macos", any(target_arch = "aarch64", target_arch = "arm64")))'.dependencies]  
+llama-cpp-sys-2 = { path = "../llama-cpp-sys-2", features=["metal"], version = "0.1.48" }
```

Comment:
what does this do?

Reply:
if the target is `macos` and `arm64`, feature `metal` is on, otherwise is off.

that is, macos user of crate `llama-cpp-2` needn't care about the arch (apple or intel) .